sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -2272,7 +2272,9 @@ public void write(TableRow value) throws Exception {
 
       public final KV<String, Long> close() throws IOException {
         channel.close();
-        return KV.of(fileName, out.getCount());
+        KV<String, Long> record = KV.of(fileName, out.getCount());
```

Comment:
I cannot put it in a try-with-resources because this is an attribute and it is used in the write method, this is the reason why I have to close it manually here.